app/src/marketplace/containers/ProductPage/SubscriberGraph.jsx

```diff
@@ -95,18 +93,11 @@ const SubscriberGraph = ({ productId, shownDays = 7 }: Props) => {
     }, [subscriptionData, shownDays])
 
     return (
-        // TODO(mariusz): MediaQuery really needed!?
-        <MediaQuery maxWidth={lg.max}>
-            {(isTabletOrMobile: boolean) => (
-                <TimeSeriesGraph
-                    graphData={graphData}
-                    shownDays={shownDays}
-                    width={isTabletOrMobile ? 380 : 540}
-                    height={200}
-                    isLoading={isLoading}
-                />
-            )}
-        </MediaQuery>
+        <TimeSeriesGraph
```

Comment:
Seems like the `width` and `height` parameters didn't do much so I removed them.